src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -124,10 +124,11 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
             allUsersStatus <- directoryDAO.isGroupMember(allUsersGroup.id, user.id, samRequestContext).unsafeToFuture() recover { case _: NameNotFoundException => false }
             tosAcceptedStatus <- tosService.getTosStatus(user.id).unsafeToFuture()
             ldapStatus <- registrationDAO.isEnabled(user.id, samRequestContext).unsafeToFuture()
+            adminEnabled <- directoryDAO.isEnabled(user.id, samRequestContext).unsafeToFuture()
           } yield {
             val enabledMap = Map("ldap" -> ldapStatus, "allUsersGroup" -> allUsersStatus, "google" -> googleStatus)
             val enabledStatuses = tosAcceptedStatus match {
-              case Some(status) => enabledMap + ("tosAccepted" -> status)
+              case Some(status) => enabledMap + ("tosAccepted" -> status) + ("adminEnabled" -> adminEnabled)
```

Comment:
Why do we only include adminEnabled in the map if tosAcceptedStatus == true?  Am I reading that correctly?

Reply:
Ah! The nuance is that tosAcceptedStatus can be Some(true) or Some(false), and adminEnabled will be added in both of these cases. If ToS is enabled, the status will be Some(_) and if it's disabled the status will be None. So with this change we're adding the adminEnabled field in cases where Tos is enabled.

Reply:
Gotcha.  I should have realized that.  Thanks for your patience.